cmd/edgectl/intercept.go

```diff
@@ -326,7 +324,6 @@ func MakeIntercept(p *supervisor.Process, out *Emitter, tm *TrafficManager, clus
 	}
 
 	manifest, err := json.MarshalIndent(&mapping, "", "  ")
-
```

Comment:
Dude. Vertical whitespace is not a bad thing. :joy:

Reply:
I blame `gofmt`! I didn't remove those on purpose.

Reply:
*!@&*!#& but whatever. :wink: